test-rabbitmq-csv-send/src/main/java/org/voltdb/bulkloader/RMQCSVSend.java

```diff
@@ -133,7 +133,9 @@ public boolean hasNext()
         @Override
         public String next()
         {
-            return this.getNextLine();
+            String nextLine = this.getNextLine();
```

Comment:
The changes look good. Just a nit - use a local variable name that is different from the member variable name.